MIGRATION_GUIDE.md

```diff
@@ -44,7 +49,7 @@ New fields:
 - added `describe_output` field that holds the response from DESCRIBE SCHEMA. Note that one needs to grant sufficient privileges e.g. with [grant_ownership](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest/docs/resources/grant_ownership) on all objects in the schema. Otherwise, this field is not filled.
 - added `parameters` field that holds the response from SHOW PARAMETERS IN SCHEMA.
 
-We allow creating and managing `PUBLIC` schemas now. When the name of the schema is `PUBLIC`, it's created with `OR_REPLACE`. We've decided this based on [#2826](https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/2826).
+We allow creating and managing `PUBLIC` schemas now. When the name of the schema is `PUBLIC`, it's created with `OR_REPLACE`. Please be careful with this operation, because you may experience data loss. The alternative is to import `PUBLIC` schema manually and then manage it with Terraform. We've decided this based on [#2826](https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/2826).
```

Comment:
Please elaborate a bit more on the data loss (i.e. REPLACE does DROP underneath, tf plan will just list this as create, link to time-travel)

Reply:
Also, did you validate that importing works in the older/currentversion?

Reply:
Added more info to the doc. Regarding import, I tested it manually and it works, but I can't setup automatic tests to import in v0.93.0. As we discussed, this may be because of separate state for versioned providers in the tests. Will create a follow-up ticket for this.